### PR TITLE
Task/ctaa 1360 configuration from server

### PR DESCRIPTION
**Pull request by falkorichter**
## Changes

create the Exposure Framework configuration based on the configuration provided by our backend.

Mapping has been live reviewed and double checked by @uluckas in a pair programming session

## Solved issues

- part of [Issue #CTAA-1360](https://tasks.pxp-x.com/browse/CTAA-1360)
- part of [Issue #CTAA-1361](https://tasks.pxp-x.com/browse/CTAA-1361)

### CODE REVIEW

**Comment by falkorichter** [comment]
depends on #114 I will update it after the review of #114 is done

File: app/src/main/res/raw/configuration.json [hunk 1]
```diff
@@ -1,9 +1,18 @@
 {
   "configuration": {
-    "warn_before_symptoms": 54,
+    "upload_keys_days": 2,
     "red_warning_quarantine": 336,
     "yellow_warning_quarantine": 168,
     "self_diagnosed_quarantine": 168,
+    "exposure_configuration": {
+      "minimum_risk_score": 1,
+      "daily_risk_threshold": 30,
```
Comment (uliluckas) [inline app/src/main/res/raw/configuration.json]:
```suggestion
      "daily_risk_threshold": 7,
```

File: app/src/main/res/raw/configuration.json [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+      "attenuation_duration_thresholds": [33, 63],
+      "attenuation_level_values": [0, 1, 2, 2, 8, 8, 8, 8],
+      "days_since_last_exposure_level_values": [1, 1, 1, 1, 1, 1, 1, 1],
+      "duration_level_values": [0, 1, 2, 3, 4, 5, 6, 7],
+      "transmission_risk_level_values": [1, 1, 1, 1, 1, 1, 1, 1]
+    },
```
Comment (uliluckas) [inline app/src/main/res/raw/configuration.json:15]: Not yet final. Please hold on for the final version

Comment (falkorichter) [reply app/src/main/res/raw/configuration.json:15]: yes, this review will keep my from finishing the PR anyways :)

**Comment by falkorichter** [comment]
As per discussion with @dusanjencik, merged this into #114

**Pull request merged**